Bottom: 501244f52681273979e3ea577e635987e94396ef
Top:    768d0728dfcd8b6bebd74bee17e75e61a197403c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 22:23:29 +0100

action.c: Remove dead code, IgnoreMotionEvents


---

diff --git a/src/action.c b/src/action.c
index d892429..e222dbf 100644
--- a/src/action.c
+++ b/src/action.c
@@ -323,7 +323,6 @@ static int defer_updates = 0;
 static int defer_needs_update = 0;
 
 static Cardinal polyIndex = 0;
-static bool IgnoreMotionEvents = false;
 static bool saved_mode = false;
 #ifdef HAVE_LIBSTROKE
 static bool mid_stroke = false;
@@ -2152,18 +2151,12 @@ EventMoveCrosshair (int ev_x, int ev_y)
       return;
     }
 #endif /* HAVE_LIBSTROKE */
-  /* ignore events that are caused by ActionMovePointer */
-  if (!IgnoreMotionEvents)
+  if (MoveCrosshairAbsolute (ev_x, ev_y))
     {
-      if (MoveCrosshairAbsolute (ev_x, ev_y))
-	{
-	  /* update object position and cursor location */
-	  AdjustAttachedObjects ();
-	  notify_crosshair_change (true);
-	}
+      /* update object position and cursor location */
+      AdjustAttachedObjects ();
+      notify_crosshair_change (true);
     }
-  else
-    IgnoreMotionEvents = false;
 }
 
 /* --------------------------------------------------------------------------- */
